test/test_transforms.py

```diff
@@ -519,6 +519,27 @@ def test_scriptmodule_TimeStretch(self):
         tensor = torch.rand((10, 2, n_freq, 10, 2))
         _test_script_module(transforms.TimeStretch, tensor, n_freq=n_freq, hop_length=hop_length, fixed_rate=fixed_rate)
 
+    def test_batch_Fade(self):
+        waveform, sample_rate = torchaudio.load(self.test_filepath)
+        fade_in_len = 3000
+        fade_out_len = 3000
+
+        # Single then transform then batch
+        expected = transforms.Fade(fade_in_len, fade_out_len)(waveform).repeat(3, 1, 1)
+
+        # Batch then transform
+        computed = transforms.Fade(fade_in_len, fade_out_len)(waveform.repeat(3, 1, 1))
+
+        self.assertTrue(computed.shape == expected.shape, (computed.shape, expected.shape))
+        self.assertTrue(torch.allclose(computed, expected))
+
+    def test_scriptmodule_Fade(self):
+        waveform, sample_rate = torchaudio.load(self.test_filepath)
+        fade_in_len = 3000
+        fade_out_len = 3000
+
+        _test_script_module(transforms.Fade, waveform, fade_in_len, fade_out_len)
```

Comment:
Can you show the code snippet and error stack that fails as you mentioned above?

Reply:
It was because to use `torch.jit.script` is necessary to:

> var = None
>  if var is None:

instead of:
> var = None
>  if var:

So it was my fault.